titan_wgs_utils/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: titan_wgs_utils
+  version: 1.10.0
+source:
+  fn: TitanCNA_1.10.0.tar.gz
+  url: https://bioconductor.riken.jp/packages/3.3/bioc/src/contrib/TitanCNA_1.10.0.tar.gz
+build:
+  number: 2
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
+  build:
+    - 'bioconductor-hmmcopy'
+    - 'bioconductor-iranges'
+    - 'bioconductor-rsamtools'
+    - 'bioconductor-genomeinfodb'
+    - 'bioconductor-snpchip'      
+    - 'r-foreach'
+    - 'r-argparse'
+    - 'r-domc'
+    - 'r >=3.1.0'
+    - perl >=5.22.0
+  run:
+    - 'bioconductor-hmmcopy'
+    - 'bioconductor-iranges'
+    - 'bioconductor-rsamtools'
+    - 'bioconductor-genomeinfodb'
+    - 'bioconductor-snpchip'
+    - 'r-foreach'
+    - 'r-argparse'
+    - 'r-domc'
+    - 'r >=3.1.0'
+    - python
+    - perl >=5.22.0
+    - numpy
+    - pandas
+test:
+  commands:
+    - '$R -e "library(''TitanCNA'')"'
+    - 'createTITANsegmentfiles.pl -h | cat'
+    - 'parse_titan_segments.py -h'
+about:
+  home: http://bioconductor.org/packages/HMMcopy/
```

Comment:
titan here please?

Reply:
shoulld it be changed in titan_wgs as well?